.github/workflows/testing.yaml

```diff
@@ -0,0 +1,58 @@
+name: testing
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  linux:
+    # We want to run on external PRs, but not on our own internal
+    # PRs as they'll be run by the push to the branch.
+    #
+    # The main trick is described here:
+    # https://github.com/Dart-Code/Dart-Code/pull/2375
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
+
+    strategy:
+      fail-fast: false
+      matrix:
+        tarantool:
+          - '1.10'
+          - '2.4'
+          - '2.5'
+          - '2.6'
+          - '2.7'
+          - '2.8'
+
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install tarantool ${{ matrix.tarantool }}
+        uses: tarantool/setup-tarantool@v1
+        with:
+          tarantool-version: ${{ matrix.tarantool }}
+
+      - name: Clone the module
+        uses: actions/checkout@v2
+
+      - name: Cache rocks
+        uses: actions/cache@v2
+        id: cache-rocks
+        with:
+          path: .rocks/
+          key: cache-rocks-${{ matrix.tarantool }}-01
+
+      - name: Install requirements
+        run: |
+          tarantoolctl rocks install luatest 0.5.0
+          CMAKE_DUMMY_WEBUI=YES tarantoolctl rocks install cartridge 2.5.1
```

Comment:
Personally, I'm not sure that it's really needed for now.
The final decision will be made by @Totktonada.

Reply:
In principle, yes, this can be added when really needed for hot-reload tests.

Reply:
Sure, it would be better to add luatest installation and so on into CI, when we'll have something to run using luatest.

Reply:
I removed this code hunk.